components/purchase_modal/purchase_modal.tsx

```diff
@@ -112,9 +121,57 @@ export default class PurchaseModal extends React.PureComponent<Props, State> {
         this.setState({processing: true, paymentInfoIsValid: false});
     }
 
+    comparePlan = (
+        <a
+            className='ml-1'
+            href='https://mattermost.com/pricing-cloud/'
```

Comment:
For compare plans like, please use: https://mattermost.com/pricing-cloud/#pricing-grid-block_5fa2028808529

My bad that I didn't include in the latest link in the Figma